geometry_msgs/Point start_position
geometry_msgs/Point goal_position

---

geometry_msgs/PoseArray traj
float64 min_curvr
bool s_flag